# HG changeset patch
# User Michael K. Johnson <http://issues.rpath.com/>
# Date 1281634877 14400
# Node ID ea2a04725873bd0900a0ff8503dab88ce4f43644
# Parent  1d030147e52fc386b03ad40c09c14d1f6bbbb2a2
Support "data.tar.lzma" in .deb files (CNY-3491)

diff --git a/NEWS.src/CNY-3491.feature b/NEWS.src/CNY-3491.feature
new file mode 100644
--- /dev/null
+++ b/NEWS.src/CNY-3491.feature
@@ -0,0 +1,1 @@
+Support "data.tar.lzma" in .deb files
diff --git a/conary/build/source.py b/conary/build/source.py
--- a/conary/build/source.py
+++ b/conary/build/source.py
@@ -618,6 +618,9 @@
                 elif debData.endswith('.xz'):
                     _uncompress = "xz -d -c"
                     actionPathBuildRequires.append('xz')
+                elif debData.endswith('.lzma'):
+                    _uncompress = "xz -d -c"
+                    actionPathBuildRequires.append('xz')
                 else:
                     # data.tar?  Alternatively, yet another
                     # compressed format that we need to add
